x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/tasks/ingest_pipelines_stats.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { LogMeta, Logger } from '@kbn/core/server';
+import type { ITelemetryEventsSender } from '../sender';
+import type { ITelemetryReceiver } from '../receiver';
+import type { TaskExecutionPeriod } from '../task';
+import type { ITaskMetricsService } from '../task_metrics.types';
+import { TelemetryCounter } from '../types';
+import {
+  createUsageCounterLabel,
+  getPreviousDailyTaskTimestamp,
+  newTelemetryLogger,
+} from '../helpers';
+import { TELEMETRY_NODE_INGEST_PIPELINES_STATS_EVENT } from '../event_based/events';
+
+const COUNTER_LABELS = ['security_solution', 'pipelines-stats'];
+
+export function createIngestStatsTaskConfig() {
+  const taskType = 'security:ingest-pipelines-stats-telemetry';
+  return {
+    type: taskType,
+    title: 'Security Solution Telemetry Ingest Pipelines Stats task',
+    interval: '24h',
+    timeout: '1m',
```

Comment:
I assume this is high enough for larger clusters? What would happen if we make it a bit higher like 5 minutes?